client/Assets/Scripts/Skills/Skill.cs

```diff
@@ -110,6 +109,18 @@ protected override void Start()
         }
     }
 
+    public ProjectileInfo GetProjectileFromSkill()
+    {
+        if (skillInfo.projectileInfo)
+        {
+            return skillInfo.projectileInfo;
+        }
+        else
+        {
+            return null;
+        }
```

Comment:
This can be simplified doing:
```suggestion
           return skillInfo.projectileInfo ? skillInfo.projectileInfo : null;

```